src/core/api/base.go

```diff
@@ -39,8 +41,11 @@ const (
 
 // the managers/controllers used globally
 var (
-	projectMgr    project.Manager
-	repositoryMgr repository.Manager
+	projectMgr         project.Manager
+	repositoryMgr      repository.Manager
+	retentionScheduler retention.Scheduler
```

Comment:
There is no retention scheduler anymore, use the pkg/scheduler instead: https://github.com/goharbor/harbor/blob/feature/tag_retention/src/pkg/scheduler/scheduler.go#L40